src/docs/product/performance/web-vitals.mdx

```diff
@@ -16,13 +18,10 @@ Frontend transactions may contain web vitals information if the browser provides
 | [First Input Delay](#first-input-delay-fid)               | Chromium, Firefox, and Safari | 300 milliseconds  |
 | [Cumulative Layout Shift](#cumulative-layout-shift-cls)   | Chromium                      | 0.25              |
 
+<Note>
 
-## Baseline Value
-
-The baseline value for each web vital is the 75th percentile of the observed data points. It is compared against a failure threshold to give a pass or fail for each web vital.
+Some web vitals such as FP, FCP, LCP, and TTFB are measured relative to the start of the transaction. Values may differ when compared to values generated with other tools such as [Lighthouse](https://github.com/GoogleChrome/lighthouse).
```

Comment:
i agree it's a little confusing, but 1. i think it was already using it before explaining it, 2. the name is written out in the table above, 3. it's just a note, if the reader doesn't understand i don't think it's too big a deal, and 4. i don't have any better ideas for where to put it. i considered putting all this stuff at the bottom since that's a more logical progression, but i also think that 90% of visitors will probably already be aware of web vitals and i don't want to bury our value-add.

Reply:
In general, we do need to introduce the terms prior to using their acronym. @imatwawana has an idea that she'll propose.

Reply:
I'd suggest adding the acronym, in parentheses, next to the spelled out term in the table before this section. For example: First Paint (FP). That should resolve the issue of having to introduce the acronyms in this paragraph.